src/uct/ib/base/ib_md.c

```diff
@@ -191,6 +197,13 @@ static ucs_config_field_t uct_ib_md_config_table[] = {
      "Maximal number of invalidated memory keys that are kept idle before reuse.",
      ucs_offsetof(uct_ib_md_config_t, ext.max_idle_rkey_count), UCS_CONFIG_TYPE_UINT},
 
+#if ENABLE_DEBUG_DATA
+    {"SIG_DEBUG_MODE", "full",
```

Comment:
Does that mean `sig_mode` doesn't get a default value if !DEBUG?
IMO having a struct member which always exists but gets initialized only on conditional-compilation is not so good. Even though currently it doesn't cause a problem (as the logic in `uct_ib_mlx5_sig_mr_init` seems robust enough to handle it), that might not be the case in future.

Reply:
added initialization
https://github.com/Mellanox/ucx/pull/42/commits/239e23e8f160c30bb57e9db665492c4c9f12d6cf#diff-975534026ea952036f6617e7a106314af680d3ca405d777ab04e0af2b427499aR1641